### DISCUSSION TITLE
Issue #249: RTDB doesn't recognize offline mode

### DISCUSSION THREAD

**Discussion by b2m9** [firebase/firebase-js-sdk#5040] [General]
It seems that there's an issue with RTDB when toggling offline mode in dev tools ([issue 249](https://github.com/firebase/firebase-js-sdk/issues/249)). The issue is open since 2017 with little/no feedback from Firebase. Can we get some information on that? Can you reproduce it on your end? Is this feature considered stable? Is it known that it doesn't work with DevTools?

Any form of feedback is highly appreciated. Thank you.

**Comment by Feiyang1** [comment]
@schmidt-sebastian

**Reply by b2m9** [reply]
Just leaving a note here that the issues hasn't been addressed yet. Any form of response in #249 would be helpful. :pray:
